LAPACKE `strict-prototypes` compiler warning for `LAPACKE_get_nancheck` prototype

https://github.com/xianyi/OpenBLAS/blob/cf46c9f84e5cfa848c8062040cd1f833dd3b9862/lapack-netlib/LAPACKE/include/lapacke.h#L12578

If compiler warning `-Werror=strict-prototypes` is set, the following message appears:

    include/lapacke.h:12578:1: error: function declaration isn’t a prototype [-Werror=strict-prototypes]

A simple fix: 

`int LAPACKE_get_nancheck(void);`

--- a/lapack-netlib/LAPACKE/include/lapacke.h
+++ b/lapack-netlib/LAPACKE/include/lapacke.h
@@ -12575,7 +12575,7 @@ lapack_int LAPACKE_zhetrs_aa_2stage_work( int matrix_layout, char uplo, lapack_i
                    
 /* APIs for set/get nancheck flags */
 void LAPACKE_set_nancheck( int flag );
-int LAPACKE_get_nancheck( );
+int LAPACKE_get_nancheck( void );
 
 #ifdef __cplusplus
 }

